pipelines/analysis_matched.yaml

```diff
@@ -8,6 +8,22 @@ tasks:
       python: |
         from metric_pipeline_tasks import PA1Task
         config.measure.retarget(PA1Task)
+  PA2:
+    class: metric_pipeline_tasks.MatchedCatalogAnalysisTask
+    config:
+      connections.package: validate_drp
+      connections.metric: PA2
+      python: |
+        from metric_pipeline_tasks import PA2Task
+        config.measure.retarget(PA2Task)
+  PF1:
+    class: metric_pipeline_tasks.MatchedCatalogAnalysisTask
+    config:
+      connections.package: validate_drp
+      connections.metric: PF1
```

Comment:
Same as above.  Think about calling it `PF1_design` instead.

Reply:
Agreed - I did that in the pipeline yaml.